dinstaller-cli/src/main.rs

```diff
@@ -1,9 +1,12 @@
 use clap::Parser;
-use std::error;
 
-use dinstaller_cli::commands::{Commands, ConfigCommands};
-use dinstaller_lib::{software, storage, users};
-use dinstaller_cli::printers::{print, Format};
+mod commands;
+mod config;
+mod printers;
+
+use commands::Commands;
+use config::run as run_config_cmd;
```

Comment:
to be honest I would not use this `use` at all and jsut use in code `config::run` as it is shorter and easier to find